cicd/Jenkins/Jenkinsfile

```
@@ -807,34 +617,7 @@ pipeline {
             }
             steps {
               script {
-                sh "rm -rf dist"
-                unstash name: "DIST-CHI"
-                def s3Uri = "${GIT_COMMIT}-chi-pr-po-ctl"
-                sh script:"""
-                  cat << 'PolicyDocument' > policy.json
-{
-   "Statement": [
-      {
-         "Effect": "Allow",
-         "Principal": "*",
-         "Action": "s3:GetObject",
-         "Resource": "arn:aws:s3:::${s3Uri}/*"
-      }
-   ]
-}
-PolicyDocument
-                """, label: "Policy.json"
-
-                jslAwsCliWrapper("s3api create-bucket --bucket ${s3Uri} --acl public-read", 'us-east-1', true)
-                jslAwsCliWrapper("s3api put-bucket-policy --bucket ${s3Uri} --policy file://policy.json", 'us-east-1', true)
-                jslAwsCliWrapper("s3api get-bucket-policy --bucket ${s3Uri}", 'us-east-1', true)
-                jslAwsCliWrapper("s3 cp ./dist s3://${s3Uri} --recursive", 'us-east-1', true)
-                jslAwsCliWrapper("s3 website s3://${s3Uri} --index-document index.html --error-document index.html", 'us-east-1', true)
-
-
-                // sh "sleep 10m"
-                // jslAwsCliWrapper("s3 rb ${s3Uri} --force", 'us-east-1', true)
-                echo "This will publish dev build to S3"
+                publishToAssetsServer()
```

```suggestion
                publishToS3()
```
Can we change the name of the function to make it more specific?

---

Just changed that